drivers/storage/portworx/component/autopilot.go

```diff
@@ -253,25 +255,37 @@ func (c *autopilot) createServiceAccount(
 }
 
 func (c *autopilot) createClusterRole() error {
+	var rules []rbacv1.PolicyRule
+	if c.k8sVersion.GreaterThanOrEqual(k8sutil.K8sVer1_26) {
```

Comment:
PSP was deprecated on 1.25 not 1.26

Reply:
Thanks for catching it. Sorry for the typo. Fixed with latest commit